frontend/src/resources/utils/get-cluster.ts

```diff
@@ -1302,7 +1304,14 @@ export function getClusterStatus(
     )
 
     if (managedCluster && HostedClusterReadyStatus?.status === 'True') {
-      mcStatus = ClusterStatus.importing
+      const stuckImport =
+        checkForCondition('HubAcceptedManagedCluster', mcConditions) &&
+        getConditionReason('HubAcceptedManagedCluster', mcConditions) === 'Error'
+      if (stuckImport) {
+        mcStatus = ClusterStatus.importfailed
```

Comment:
Is there a message from one of the conditions that you could put into `statusMessage` so the user has some more information on what the problem is here?

Reply:
I could add this, is it too verbose?
![image](https://user-images.githubusercontent.com/15898988/226034646-feea5954-f918-42a7-b49c-b56d4f0947fc.png)

Reply:
I think it's much better than having no explanation for the failure You can see what the problem is if you read it carefully.